packages/paste-core/components/blockquote/src/BlockquoteSource.tsx

```diff
@@ -0,0 +1,64 @@
+import { Anchor } from "@twilio-paste/anchor";
+import { Box, type BoxProps } from "@twilio-paste/box";
+import { Text } from "@twilio-paste/text";
+import type { HTMLPasteProps } from "@twilio-paste/types";
+import React from "react";
+
+import { BlockquoteContext } from "./BlockquoteContext";
+
+export interface BlockquoteSourceProps extends HTMLPasteProps<"div"> {
+  /**
+   * Overrides the default element name to apply unique styles with the Customization Provider
+   * @default 'BLOCKQUOTE_SOURCE'
+   * @type {BoxProps['element']}
+   * @memberof BlockquoteSourceProps
+   */
+  element?: BoxProps["element"];
+
+  /**
+   * The author of the quote
+   * @type {string}
+   * @memberof BlockquoteSourceProps
+   */
+  author: string;
+
+  /**
+   * The source of the quote
+   * @type {string}
+   * @memberof BlockquoteSourceProps
+   */
+  source?: string;
+}
+
+export const BlockquoteSource = React.forwardRef<HTMLDivElement, BlockquoteSourceProps>(
+  ({ element = "BLOCKQUOTE_SOURCE", author, source, ...props }, ref) => {
+    const { url } = React.useContext(BlockquoteContext);
+
+    return (
+      <Box marginTop="space30" as="p" element={element} {...props} ref={ref}>
```

Comment:
question: why the mix of box as p and text as span here?

Reply:
I think span is right in this case otherwise they wuld be on separate line. The author and source styling are different